pkg/curatedpackages/curatedpackages.go

```diff
@@ -51,6 +52,74 @@ func GetVersionBundle(reader Reader, eksaVersion string, spec *v1alpha1.Cluster)
 	return versionsBundle, nil
 }
 
+func formatYamlLine(space, key, value string) string {
+	if value == "" {
+		return ""
+	}
+	return space + key + ": " + value + "\n"
+}
+
+func formatImageResource(resource *v1alpha1.ImageResource, name string) (result string) {
+	if resource.CPU != "" || resource.Memory != "" {
+		result = "    " + name + ":\n"
+		result += formatYamlLine("      ", "cpu", resource.CPU)
+		result += formatYamlLine("      ", "memory", resource.Memory)
+	}
+	return result
+}
+
+func formatCronJob(cronJob *v1alpha1.PackageControllerCronJob) (result string) {
+	if cronJob != nil {
+		result += "cronjob:\n"
+		result += formatYamlLine("  ", "digest", cronJob.Digest)
+		result += formatYamlLine("  ", "repository", cronJob.Repository)
+		result += formatYamlLine("  ", "suspend", strconv.FormatBool(cronJob.Disable))
+		result += formatYamlLine("  ", "tag", cronJob.Tag)
+	}
+	return result
+}
+
+func formatResources(resources *v1alpha1.PackageControllerResources) (result string) {
+	if resources.Limits.CPU != "" || resources.Limits.Memory != "" ||
+		resources.Requests.CPU != "" || resources.Requests.Memory != "" {
+		result += "  resources:\n"
+		result += formatImageResource(&resources.Limits, "limits")
+		result += formatImageResource(&resources.Requests, "requests")
+	}
+	return result
+}
+
+// GetPackageControllerConfiguration returns the default kubernetes version for a Cluster.
+func GetPackageControllerConfiguration(clusterSpec *v1alpha1.ClusterSpec) (result string, err error) {
```

Comment:
I would recommend moving this method to pkgControllerClient since functionally this method is dealing with a packagecontroller.

Reply:
Done.